app/components/recirculation-prefooter.js

```diff
@@ -56,6 +88,17 @@ export default Component.extend(
 							'LI'
 						);
 					});
+
+					if (this.get('shouldShowPlista')) {
+						this.fetchPlista()
+							.then(this.mapPlista)
+							.then((item) => {
+								if (item.thumbnail) {
+									this.notifyPropertyChange('items');
```

Comment:
I think it should be called below the setter. Even if it doesn't matter and Ember handles it anyway, it will be easier to reason about.